src/infrastructure/navigation/account.navigation.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import { createStackNavigator } from "@react-navigation/stack";
+
+import { LoginScreen } from "../../features/account/screens/login.screen";
+import { RegisterScreen } from "../../features/account/screens/register.screen";
+import { AuthenticationsScreen } from "../../features/account/screens/authentications.screen";
+import { LoadingScreen } from "../../features/account/screens/loading.screen";
+import { ForgotPasswordScreen } from "../../features/account/screens/forgot-password.screen";
+
+const AccountStack = createStackNavigator();
+
+export const AccountNavigator = () => (
+  <AccountStack.Navigator headerMode="none">
+    {/* <AccountStack.Screen
```

Comment:
why does it comment out? Can you add the documentation for this code snippet?

Reply:
I commented this out because I might use this or delete this depends on when we implement firebase